src/main/java/org/jabref/gui/icon/IconTheme.java

```diff
@@ -359,7 +359,12 @@ public enum JabRefIcons implements JabRefIcon {
         REMOVE_TAGS(MaterialDesignC.CLOSE),
         ACCEPT_LEFT(MaterialDesignS.SUBDIRECTORY_ARROW_LEFT),
         ACCEPT_RIGHT(MaterialDesignS.SUBDIRECTORY_ARROW_RIGHT),
-        MERGE_GROUPS(MaterialDesignS.SOURCE_MERGE);
+        MERGE_GROUPS(MaterialDesignS.SOURCE_MERGE),
+        THUMB_UP(MaterialDesignT.THUMB_UP),
+        THUMB_DOWN(MaterialDesignT.THUMB_DOWN),
+        THUMB_UP_OUTLINE(MaterialDesignT.THUMB_UP_OUTLINE),
+        THUMB_DOWN_OUTLINE(MaterialDesignT.THUMB_DOWN_OUTLINE);
```

Comment:
I did not find any usage in the code. Maybe, this can be moved to another branch (and not included here?)